### PR TITLE
Add entitlements

### PR DESCRIPTION
**Pull request by vikaspotluri123**

### CODE REVIEW

File: schools/aggiecorps.yml [hunk 1]
```diff
@@ -8,6 +8,10 @@ theme:
   background: 'https://static.gradebook.app/sYbR9JXKTI/aggiecorps.jpg'
 gpa: true
 partner: true
```
Comment (ramsays) [inline schools/aggiecorps.yml:10]: Do we still have areas where `partner` is used?

File: schools/aggie.yml [hunk 2]
```diff
@@ -2,6 +2,8 @@ version: 0.1
 name: Aggie
 label: TAMU - Texas A&M University
 studentEmailDomain: tamu.edu
+entitlements:
+  course-search: false
```
Comment (ramsays) [inline schools/aggie.yml:6]: Do we currently restrict course search to the Corps?

File: schools/aggiecorps.yml (hunk 1 above)
Comment (vikaspotluri123) [reply schools/aggiecorps.yml:10]: Yes, this is an API change so it needs to be done incrementally

File: schools/aggie.yml (hunk 2 above)
Comment (vikaspotluri123) [reply schools/aggie.yml:6]: Noe

File: schools/aggiecorps.yml (hunk 1 above)
Comment (ramsays) [reply schools/aggiecorps.yml:10]: Sounds good, as long as we note it somewhere to clean it up later 👍

File: schools/aggie.yml (hunk 2 above)
Comment (ramsays) [reply schools/aggie.yml:6]: So are we intending to not allow course search on aggie?

Comment (vikaspotluri123) [reply schools/aggie.yml:6]: That would be a typo 😄

**Review by ramsays — APPROVED** [review]

**Review by joshcosta — APPROVED** [review]

**Pull request merged**